docs/guides/migration-recipe.md

```diff
@@ -99,9 +99,9 @@ index 68387c9..7a8ace1 100644
     CODENAME=$(cat /etc/os-release | grep VERSION_CODENAME | cut -d '=' -f 2)
    
     echo "Adding the SaltStack repository key for $UBUNTU_VERSION $CODENAME ($ARCH)..."
-    sudo curl -fsSL -o /etc/apt/keyrings/salt-archive-keyring-2023.gpg https://repo.saltproject.io/salt/py3/ubuntu/$UBUNTU_VERSION/$ARCH/SALT-PROJECT-GPG-PUBKEY-2023.gpg
+    sudo curl -fsSL -o /etc/apt/keyrings/salt-archive-keyring-2024.gpg https://packages.broadcom.com/artifactory/api/security/keypair/SaltProjectKey/public
     echo "Adding the SaltStack repository for $UBUNTU_VERSION $CODENAME ($ARCH)..."
-    echo "deb [signed-by=/etc/apt/keyrings/salt-archive-keyring-2023.gpg arch=$ARCH] https://repo.saltproject.io/salt/py3/ubuntu/$UBUNTU_VERSION/$ARCH/latest $CODENAME main" | sudo tee /etc/apt/sources.list.d/salt.list
+    echo "deb [signed-by=/etc/apt/keyrings/salt-archive-keyring-2024.gpg arch=$ARCH] https://packages.broadcom.com/artifactory/saltproject-deb/ stable main" | sudo tee /etc/apt/sources.list.d/salt.list
```

Comment:
Should the pinning command be part of this doc now also?

Reply:
[9acd4ba](https://github.com/python/psf-salt/pull/524/commits/9acd4bae92be781af8bbcff78296de87c447a52b)